src/js/themes/base.d.ts

```diff
@@ -809,14 +833,9 @@ export interface ThemeType {
       margin?: MarginType;
     };
     options?: {
-      container?: {
-        align?: string,
-        pad?: string,
-      },
-      text?: {
-        margin?: MarginType,
-      },
-    },
+      container?: PropsOf<typeof Box>;
```

Comment:
I might not be fully understanding what `PropsOf` is doing here, but shouldn't it allow only `align` and `pad` instead of a whole type of Box?

Reply:
I think you’re right in assuming that `PropsOf<typeof Box>` is grabbing all of the props of `Box` and allowing them here. I wonder if we need to also do a `Pick` here:

```TypeScript
Pick<PropsOf<typeof Box>, 'align' | 'pad'>;
```

Reply:
Or, we can just import PadType from "grommet/utils", and use it instead.
```
import { PadType } from 'grommet/utils';
```
Sounds right?

Reply:
Sounds good, I can update that to one of the above... I think I got the idea that it could take any of the Box props from the sentence in the docs here, although now I do see the example only mentioned align and pad:

![image](https://user-images.githubusercontent.com/15009735/76477183-e5582180-63da-11ea-82f6-af0269dcd604.png)

Reply:
my bad, I think you are right! I checked the code and select.options.container can accept any Box prop, align and pad are just an example.

Reply:
Cool, no worries! It's back to using PropsOf again